### ISSUE TITLE
[HIPRI] China, PLAN, Type 093B & Type 095 SSGN, further upgrade

### ISSUE THREAD

**Issue by LuigiP57** [PygmalionOfCyprus/cmo-db-requests#4762]
### DB Selector

DB3K

### Affected DBID(s)

submarine 665 676

### Summary of Changes

Well, since the DB team values reviewed units so much, I will now submit the relevant ticket.

Note: I'm going to rely heavily on the SDF's analysis, and while I (and I'm sure others) can't provide exact, factual information, the opinions I offer are all enough reasonable (both in reality and in the game). I hope you can give me enough trust.

We must admit that **most OSINT or think tank reports** have seriously **underestimated and made mistakes** in the estimation of PLAN submarines development (including I was misled a lot before).

Thus, For Type 093B (Submarine 665):

- [x] Change its signatures to the **688i/Improved Los Angeles (Submarine 685)** level.
- [x] Slightly increase its max speed from 30kt to **32kt**. Better than Type 093/A, better performance.
- [x] Change its VLS number to **24**. Just **copy the current 095 VLS mount and increase 6 cells**, carry YJ-18, CJ-10 and YJ-21, this is the easiest way.
- [x] Add **Shrouded propulsor, Nonmegnetic hull and Shock resistant** to its properties. Prior to 093B, the 093/A series alone showed signs of **multiple iterations of upgrades**, reflected in the fact that **each 093/A was different from one another**.
- [x] Add **Generic SIGINT Advanced 2020s**. Many of the submarines in the game have this kind of device to detect MPA threat (AN/BLQ-10 SIGINT for exp). From the known models, it is clear that there is a similar device among its complex and varied masts

For more advanced Type 095 (Submarine 676):

- [x] Create a **Chinese VPM** mount, **3x YJ-21 in 1x cell**
- [x] Replace the 18x Single VLS tube with **12x VPM cells (2 row, each row with 6 cells)**. The ROKN Dosan-Ahn-changho SSB/SSG is 8x or 10x VLS but it's way smaller. While SSBN like 955A Borei is 16x VLS which is too many for an SSGN.
- [x] Change its signatures to the **SSN 774 (Submarine 40)** level. I remember I mentioned this before, if you've changed it, please ignore it and mark it completed.
- [x] Slightly increase its max speed from 32kt to **34kt**. Better than Type 093B, new reactor and advanced propulsor.
- [x] Add **Advanced propulsor** to its properties.
- [x] Add **Generic SIGINT Advanced 2020s**

Lastly, their **sensor tech level** is quiet low which is definately unreasonable. However, yeah, we have no information of their state-of-art sensors so I have no plans to change them at the moment, of course, if you would like, that would be great.

I expect you will be more or less skeptical, but there is no denying that China is not the Soviet Union, and it now leads the world in industrial output and innovation, and its military power growth is obvious to all.

### Sources

https://www.sinodefenceforum.com/t/09iii-09iv-093-094-nuclear-submarine-thread.6883/page-350
https://www.sinodefenceforum.com/t/09iii-09iv-093-094-nuclear-submarine-thread.6883/page-353
https://www.sinodefenceforum.com/t/09iii-09iv-093-094-nuclear-submarine-thread.6883/page-333
https://www.sinodefenceforum.com/t/09iii-09iv-093-094-nuclear-submarine-thread.6883/page-261
https://www.janes.com/osint-insights/defence-news/sea/china-launches-second-possible-type-093b-hull
For comparison:
![image](https://github.com/user-attachments/assets/3fb978d4-a1a1-4d84-9251-dae4e623afdc)
![image](https://github.com/user-attachments/assets/f94af410-cad8-420e-8b63-ffffa119e3ac)
093/A variants (though they are all wrongly designated as 093B or 095):
https://www.navalanalyses.com/2018/04/infographics-31-peoples-liberation-army.html
![image](https://github.com/user-attachments/assets/99076dd6-4f4a-4f52-a35e-aa5f782e2172)
credit to Naval Analyses

https://www.sinodefenceforum.com/t/09v-09vi-095-096-nuclear-submarine-thread.8228/page-128
https://www.sinodefenceforum.com/t/09v-09vi-095-096-nuclear-submarine-thread.8228/page-129
According to the paper's simulations, it appears that the VPM can also be mounted on the front of the submarine, suggesting that the hull diameter of 095 May be large enough to accommodate a large number of VPMs.
![image](https://github.com/user-attachments/assets/f291ca92-4543-47c9-9449-47572ccb8aa5)
Advanced propulsor
https://www.scmp.com/news/china/science/article/3195922/chinas-submarines-can-be-quieter-more-powerful-new-pump-jet
![image](https://github.com/user-attachments/assets/d182a44f-8e54-4aa9-8f25-e85ed31ed76a)
https://www.sinodefenceforum.com/t/09v-09vi-095-096-nuclear-submarine-thread.8228/page-130
![image](https://github.com/user-attachments/assets/5be7cb07-89f6-45c0-a4a9-19bd3fa6803b)
https://www.nextbigfuture.com/2017/07/china-all-electric-rim-driven-shaftless-ultraquiet-submarine-propulsion.html

More about VPM and VLS tube:
#4750

**Comment by Luigi-II** [comment]
Comparison of Type 093A and B
https://www.sinodefenceforum.com/t/09iii-09iv-093-094-nuclear-submarine-thread.6883/page-358
![image](https://github.com/user-attachments/assets/5b3e35d9-5dad-4c0a-8664-71332bb8ab84)

**Comment by LuigiP57** [comment]
@Jkryos Regarding the number of vertical launch tubes on Type 093B submarines, new pictures and information from Chinese internet sources suggest that it should have 24 tubes. The 3x6 model is a scam by CSSC. Well, this is understandable.

https://www.navalnews.com/naval-news/2024/10/chinese-type-09iiib-nuclear-powered-submarine-surfaces-in-image/#prettyPhoto

> A related variation of this SSN design appeared in[ a video presentation of the Harbin Engineering University](https://www.bilibili.com/video/BV1rp4y1E752/) in August 2023. A graphic of the design on a wall in the University exhibition showed an overall identical SSN. The major distinction of the boat is a visible VLS-row of eight launchers. The number of rows is not clearly visible but may suggest up to 24 launchers for VLS-ordnance.

https://www.sinodefenceforum.com/t/09iii-09iv-093-094-nuclear-submarine-thread.6883/page-386

**Comment by Jkryos** [comment]
For the Type 093B stealth characteristics. I have done a fair bit of reading on the topic, and while I agree there is a clear trend of improved quieting it seems they have stagnated a little bit in the development of Anechoic paneling around the time of Bs development. Because of this I have dropped the passive sonar values to that of the LA class, the new engine improvements of the B likely also included increased quieting. I have left the active sonar at its current level.

**Comment by Jkryos** [comment]
For the same reason I disagreed with the sound changes on the 093B, I agree with the changes on the 095. There is a lot of evidence of a step up in stealth. As for the sensors, as you mention it is all speculations. That being said there is a clear evolutionary line of development from the French/Italian sonars they imported in the 80s before the sanctions to now. It seems like there is a decline return on investment development wise, with no major _public_ leaps forward since the early '00s. Updated DB3K 509.
